pkg/apply/mutator/apply_time_mutator.go

```diff
@@ -67,19 +79,27 @@ func (atm *ApplyTimeMutator) Mutate(obj *unstructured.Unstructured) (bool, strin
 			return mutated, reason, fmt.Errorf("failed to get source resource (%v): %w", sourceRef, err)
 		}
 
-		sourceID := object.UnstructuredToObjMetaOrDie(sourceObj)
-		klog.V(5).Infof("source resource %s/%s: %#v", sourceID.Namespace, sourceID.Name, sourceObj)
+		if klog.V(4).Enabled() {
+			yamlBytes, err := yaml.Marshal(sourceObj)
```

Comment:
I often define a function called `DebugJSON` or something like that that formats the object or returns a string representation of the error message.  You normally want to avoid changing the code flow based on logging, so you don't want to return.

Reply:
Good catch.
Fixed to log the error.